scikitonx/operator_converters/CalibratedClassifierCV.py

```diff
@@ -0,0 +1,289 @@
+    #         V                          V                                      |
+    #  exp_result [M, 1]  ...    exp_result [M, 1]                              |
+    #         |                          |                                      |
+    #         V                          V                                      |
+    #       ADD <------- unity -------> ADD                                     |
+    #         |                          |                                      |
+    #         V                          V                                      |
+    #  denominator [M, 1]  ...   denominator [M, 1]                             |
+    #         |                          |                                      |
+    #         V                          V                                      |
+    #        DIV <------- unity ------> DIV                                     |
+    #         |                          |                                      |
+    #         V                          V                                      |
+    # sigmoid_predict_result [M, 1] ... sigmoid_predict_result [M, 1]           |
+    #         |                          |                                      |
+    #         '-----.--------------------'                                      |
+    #               |-----------------------------------------------------------'
+    #               |
+    #               V
+    #            CONCAT -> concatenated_prob [M, C]
+    #                          |
+    #        if  C = 2         |  if C != 2
+    #      .-------------------'-----------------------------------------.-------------------.
+    #      |                                                             |                   |
+    #      V                                                             |                   V
+    # ARRAYFEATUREEXTRACTOR <- col_number [1]                            |             REDUCESUM
+    #                   |                                                |                   |
+    #                   '--------------------------------.               V                   V
+    # unit_float_tensor [1] -> SUB <- first_col [M, 1] <-'              DIV <----------- reduced_prob [M]
+    #                           |                                        |
+    #                           V                                        |
+    #                         CONCAT                                     |
+    #                           |                                        |
+    #                           V                                        |
+    #                        class_prob_tensor [M, C] <------------------'
+
+    base_model = model.base_estimator
+    op_type = sklearn_operator_name_map[type(base_model)]
+    n_classes = len(model.classes_)
+    prob_name = [None] * n_classes
+
+    this_operator = scope.declare_local_operator(op_type)
```

Comment:
Where does this operator got converted? This works because the our graph traversing algorithm dynamically extract the ready-to-be-converted operator from all scopes. However, declaring new operators (i.e., change our internal IR) while converting that IR doesn't look very safe. I am not asking you to change this immediately, but for a better quality of code, there could be a working item in the future.

Reply:
Okay, I'll create a work item for this.

Reply:
https://aiinfra.visualstudio.com/Lotus/_workitems/edit/2226